app/src/routes/api/v1/user.router.js

```diff
@@ -72,7 +72,14 @@ class UserRouter {
             return;
         }
         ctx.request.body._id = mongoose.Types.ObjectId(ctx.request.body.loggedUser.id);
-        const userCreate = await new User(ctx.request.body).save();
+        const user = new User(ctx.request.body);
+        const errors = user.validateSync();
+        if (errors) {
+            logger.error(errors.message);
```

Comment:
logger.error should be used for an application error. In this case, it's not an application error, it's most likely a "user didn't provide enough info" error, but the app continues to behave as normal, so use logger.info instead